g3proxy/docker/lua.alpine.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM nickblah/lua:5.4-luarocks-alpine AS lua
+RUN apk add --no-cache build-base
+# install lua modules here.
+# dkjson is just an example.
+# beware: lua's c modules (e.g. cjson)
+# need to be statically linked
+# and won't work when installing in
+# alpine via luarocks. 
+RUN luarocks install dkjson
+
+FROM rust:alpine AS builder
+WORKDIR /usr/src/g3
+COPY . .
+RUN apk add --no-cache musl-dev cmake capnproto-dev openssl-dev openssl-libs-static c-ares-dev c-ares-static lua5.4-dev pkgconfig
```

Comment:
openssl-libs-static and c-ares-static can be removed

Reply:
my bad, again